### PR TITLE
Added a VERSION file and updated scripts to use it

### PR DESCRIPTION
**Pull request by dmitri-d**
Introduces a single place to store/update version information -- no longer need to update it in multiple shell scripts.

### CODE REVIEW

**Comment by dmitri-d** [comment]
@tvieira, @bdecoste, @knrc, @brian-avery : what do you think?

File: istio-proxy/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+maistra-0.9
```
Comment (brian-avery) [inline istio-proxy/VERSION:1]: Wonder if we can rely on the current branch name. i.e.:

git branch | grep \* | cut -d ' ' -f2

**Review by brian-avery — APPROVED** [review]

File: istio-proxy/VERSION (hunk 1 above)
Comment (dmitri-d) [reply istio-proxy/VERSION:1]: We could, but I wanted to have branch name/version set explicitly: a pre-build check if you want.

**Comment by tvieira** [comment]
I honestly can't see a good solution for that yet. What bothers me with having the VERSION file is that we can have a `maistra-0.10` branch and its VERSION file with `maistra-0.9` value. At one time this is going to be fixed, but until that it would be out of sync.

I would imagine that a `VERSION` file would only work if we create it at build time and pack it within a tarball deliverable or within a container image. Within a version control this file will create confusion. That's my 2c.

File: istio-proxy/VERSION (hunk 1 above)
Comment (brian-avery) [reply istio-proxy/VERSION:1]: I can see a need for the ability to override to create test/debug versions based off of a different branch, but in terms of a release, I think we probably always want to make sure that the branches match up. I could go either way on this.

**Comment by dmitri-d** [comment]
VERSION file here contains the version that we want to build. This data will be stored and tagged in git, and possibly used later to build an older version of the proxy. I think for this particular application, it's ok (and necessary) to have the content of VERSION to be out of sync with the latest branch.

**Comment by brian-avery** [comment]
This was implemented using update.sh

**Pull request closed without merging**